src/rpc/rawtransaction.cpp

```diff
@@ -192,11 +195,34 @@ UniValue getrawtransaction(const JSONRPCRequest& request)
             + HelpExampleCli("getrawtransaction", "\"mytxid\"")
             + HelpExampleCli("getrawtransaction", "\"mytxid\" true")
             + HelpExampleRpc("getrawtransaction", "\"mytxid\", true")
+            + HelpExampleCli("getrawtransaction", "\"myblockhash:mytxid\"")
+            + HelpExampleCli("getrawtransaction", "\"myblockhash:mytxid\", true")
         );
 
     LOCK(cs_main);
 
-    uint256 hash = ParseHashV(request.params[0], "parameter 1");
+    uint256 hash;
+    int64_t blockHeight = -1;
+    bool inMainChain = true;
+    auto p = request.params[0].get_str().find(':');
```

Comment:
This functionality could come in useful.

As for the API I prefer to not do any string combining/parsing here, this makes the API less clean to work with at least in my experience. I'd prefer to add an optional (can be null) `fromBlock` argument.